x-pack/plugin/ilm/src/main/java/org/elasticsearch/xpack/indexlifecycle/IndexLifecycleRunner.java

```diff
@@ -323,7 +325,8 @@ static ClusterState moveClusterStateToErrorStep(Index index, ClusterState cluste
             .get(LifecycleSettings.LIFECYCLE_NAME_SETTING.get(idxMeta.getSettings()));
         XContentBuilder causeXContentBuilder = JsonXContent.contentBuilder();
         causeXContentBuilder.startObject();
-        ElasticsearchException.generateThrowableXContent(causeXContentBuilder, ToXContent.EMPTY_PARAMS, cause);
+        ElasticsearchException.generateThrowableXContent(causeXContentBuilder,
+            new ToXContent.MapParams(Collections.singletonMap(REST_EXCEPTION_SKIP_STACK_TRACE, "false")), cause);
```

Comment:
Should we make this a constant since it's always the same?

Reply:
I don't think we need to, since we don't expect to ever change it? And the JVM caches String literals for performance